src/canvas/Canvas.spec.ts

```diff
@@ -3,9 +3,138 @@ import { Rect } from '../shapes/Rect';
 import { IText } from '../shapes/IText/IText';
 import '../shapes/ActiveSelection';
 
-import { describe, expect, test, vi } from 'vitest';
+import { afterEach, beforeEach, describe, expect, it, test, vi } from 'vitest';
+import { config } from '../config';
+import type {
+  FabricObject,
+  MultiSelectionStacking,
+  TPointerEvent,
+} from '../../fabric';
+import {
+  ActiveSelection,
+  Circle,
+  classRegistry,
+  getFabricDocument,
+  Group,
+  Path,
+  version,
+} from '../../fabric';
+
+vi.mock('../util/misc/objectEnlive', async (importOriginal) => ({
+  ...(await importOriginal()),
+  loadImage: vi.fn().mockImplementation(() => {
```

Comment:
one test was loading the image in qunit, and it wasn't working in vitest like this since this is running in jsdom
so I did it like this
we can restore real image load if we eventually move to browser tests in vitest, but for now I think this is good enough